# motor speed in angular vel (rad/sec), assuming 4 wheels. 
float64 tl_speed
float64 tr_speed
float64 bl_speed
float64 br_speed